ansible/install_neutron-api.yaml

```diff
@@ -25,6 +29,29 @@
     - "ocp/neutron-api/2"
     - "ocp/neutron-api/3"
 
+  - name: get ovn connection ips
+    shell: >
+      oc get pod -n openstack -o wide | grep ovn-ovsdb | awk '{print $6}'
```

Comment:
Unfortunately this isn't correct. We need to use the service IPs here. We can probably use a very similar construct, though, this is quite neat.

One other thing, we should add this to ovn rather than neutron-api, and call the config map something related to ovn. I was going to use it for northd, too.